packages/apps/src/utils/activity-utils.ts

```diff
@@ -0,0 +1,16 @@
+import { ActivityBuilder, ActivityParams, isActivityBuilder, MessageSendActivityBuilder } from '@teams.sdk/api';
+import { Card, isCard } from '@teams.sdk/cards';
+
+export function toActivityParams(activity: ActivityParams | string | ActivityBuilder | Card): ActivityParams {
```

Comment:
it might be best for this to exist in the `api` package where the `ActivityParams` are, since the other seem to exist near their respective types.